dashboard/views/users.ejs

```diff
@@ -71,11 +72,16 @@
                                 <a href="#" onclick="deleteMultipleUsers(); return false;" class="btn btn-round" data-l10n-id="multidelete" title="Delete Multiple"><i class="material-icons text-muted">delete_forever</i></a>
                             </div>
                             <div class="btn pull-right btn-round" onclick="downloadUsers()" data-l10n-id="ExportUsers" id="users-exportusers">Export Users</div>
-                            <script>
+                            <script>                        
+                                var roleValue = document.getElementById('user-type-select2').value;
+                                var usernameValue = document.getElementById('username').value;
+                                var classroomValue = document.getElementById('classroom_select').value; 
+                                classroomValue = classroomValue == '' ? 'undefined' : classroomValue;                          
+                                usernameValue = usernameValue == '' ? 'undefined' : usernameValue;                            
                                 function downloadUsers(event) {
                                     $.ajax({
                                             type: "GET",
-                                            url: "/dashboard/users/export",
+                                            url: `/dashboard/users/export/${roleValue}/${usernameValue}/${classroomValue}`,
```

Comment:
Instead of making these parameters part of the URL, it'll be better group them in a query.

Reply:
Sure, that makes more sense.